### PR TITLE
[HMC-14] - Slicing Card Partners

### PR DESCRIPTION
**Pull request by tjoetmuty**
image

### CODE REVIEW

File: src/features/Main/components/CardPartner/index.tsx [hunk 1]
```diff
@@ -8,18 +8,30 @@ import {
 import { FC } from 'react';
 
 const CardPartners: FC = () => {
+  const Images = [
+    { src: 'efishery.png' },
+    { src: 'icehouse.png' },
+    { src: 'learning.png' },
+    { src: 'shipper.png' }
+  ];
```
Comment (mpsalunggg) [inline src/features/Main/components/CardPartner/index.tsx:16]: set to constants folder, and create one file like images.ts on the constants, paste the array @tjoetmuty

**Comment by mpsalunggg** [comment]
@tjoetmuty pull yang terbaru dari development

File: src/features/Main/components/CardPartner/Images.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const logoPartners: ImageData[] = [
+  { src: 'shipper.png' },
+  { src: 'efishery.png' },
+  { src: 'icehouse.png' },
+  { src: 'learning.png' }
+];
```
Comment (akwanmaroso) [inline src/features/Main/components/CardPartner/Images.tsx]: for better SEO performance, please add a property `alt`  and included that on img tag

additionally, each feature can add a constants folder to put the constant data like this

**Review by mpsalunggg — APPROVED** [review]
Goods

File: src/features/Main/components/CardPartner/index.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          <CardTitle className="text-[#00CCFF] flex justify-center text-center">
+            Learning From Our Expert Tech
+          </CardTitle>
+          <CardDescription className="text-[#0F3665] max-w-[80%] md:max-w-full mx-auto md:mx-0 text-center flex justify-center">
+            Take the opportunity to learn from experts from tech companies
+          </CardDescription>
+        </CardHeader>
+        <CardContent className="flex flex-wrap gap-4 justify-center">
+          {logoPartners.map((item, index) => (
+            <img
```
Comment (vickyadrii) [inline src/features/Main/components/CardPartner/index.tsx:25]: bcs we are using next.js, i'd prefer to change the tag `img` to `Image` from `Next/image` 😉

Comment (mpsalunggg) [reply src/features/Main/components/CardPartner/index.tsx:25]: @tjoetmuty yapss using next/image

**Review by vickyadrii — APPROVED** [review]

**Review by vickyadrii — APPROVED** [review]

**Pull request merged**